src/bin/cargo/cli.rs

```diff
@@ -640,17 +640,17 @@ See '<cyan,bold>cargo help</> <cyan><<command>></>' for more information on a sp
                 .value_parser(clap::builder::ValueParser::path_buf()),
         )
         .arg(
-            flag("frozen", "Require Cargo.lock and cache to be up-to-date")
+            flag("locked", "Ensure the exact dependency versions in Cargo.lock is used")
                 .help_heading(heading::MANIFEST_OPTIONS)
                 .global(true),
         )
         .arg(
-            flag("locked", "Require Cargo.lock to be up-to-date")
+            flag("offline", "Run without accessing the network")
                 .help_heading(heading::MANIFEST_OPTIONS)
                 .global(true),
         )
         .arg(
-            flag("offline", "Run without accessing the network")
+            flag("frozen", "Equivalent to specifying both --locked and --offline")
```

Comment:
> This is an attempt to remove the up-to-date mentioned in --locked.
Up-to-date is quite confusing, especially when MSRV resolution is out,
dependency version may lag behind more often.

I actually prefer the original wording for `--locked`.  Its not saying its up-to-date as in `cargo update` but that it is synchronized with `Cargo.toml`.  

With the new wording, it isn't clear that adding a new dependency will fail with `--locked` (there isn't a version in `Cargo.lock`, so its undefined according to this text(

Also, `require` makes it sound like it will fail while `ensure` makes it sound like it could fail or it could correct things (ie the opposite of `--locked`).

Reply:
> Its not saying its up-to-date as in cargo update but that it is synchronized with Cargo.toml.

That's the problem. Up-to-date can be interpreted differently. I was there and also confused until I learnt how `--locked` works internally in Cargo. Maybe something like

"Enforce only use of dependency versions captured in Cargo.lock"

Still not too happy with it.

Reply:
Some ideas for brainstorming
```
Require Cargo.lock be up-to-date with Cargo.toml
Require all dependency versions to be specified in Cargo.lock
Require Cargo.lock to specify every dependency version
Fail if Cargo.lock would change
```

Reply:
> Fail if Cargo.lock would change


Neat! Like this one. Updated this and the man pages.

Reply:
Sounds good. Updated per suggestion

Assert is also a common and strong enough word for this purpose.